### PR TITLE
New package: alsa-oss-1.0.28

### PR DESCRIPTION
**Pull request by KeepBotting**
The `alsa-oss` package provides the `aoss` script which is how ALSA maintains compatibility with the old OSS driver. This is useful to coax audio from very old programs that are only aware of OSS, and will not function properly under ALSA or PulseAudio.

### CODE REVIEW

File: srcpkgs/alsa-oss/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'alsa-oss'
+pkgname=alsa-oss
+version=1.0.28
+revision=1
+build_style=gnu-configure
+depends="alsa-lib"
+hostmakedepends="alsa-lib-devel"
```
Comment (Vaelatern) [inline srcpkgs/alsa-oss/template]: `makedepends` not `hostmakdepends`.
The `depends` line is also useless, please remove it.

**Comment by Vaelatern** [comment]
only one commit. git push -f   to make the changes.

```
case "${XBPS_TARGET_MACHINE}" in
        *-musl) broken="https://travis-ci.org/voidlinux/void-packages/jobs/271818539";;
esac
```

**Comment by KeepBotting** [comment]
Done. I used `git push -f` but all three commits still show for me.

**Comment by Vaelatern** [comment]
Fix xlint error? https://travis-ci.org/voidlinux/void-packages/jobs/271823010
`git rebase -i HEAD~4` after you have committed the above change, then `git push -f`

**Comment by Vaelatern** [comment]
In the interactive rebase you will see the option to `fixup` commits. Take it. All of these commits in this case should be one commit with the `New package:` commit message.

**Comment by Vaelatern** [comment]
feel free to practice this git stuff. Makes maintainers jobs easier, and good commits just radiate "I read the documentation and I know the system here," which is nice to see. I'm going to merge this now.

**Comment by KeepBotting** [comment]
Sorry. I'm flailing around in vim at the moment trying to figure out how to rebase properly. I'll practice that and also remember to xlint before every commit. Thanks!

**Pull request merged**